pmm-app/tests/verifyAWSRDSMySQLInstance_test.js

```diff
@@ -1,15 +1,16 @@
-Feature('To verify monitoried AWS RDS Db instance');
+Feature('Monitoring AWS RDS MySQL DB');
 
 Before((I, loginPage) => {
-    I.amOnPage(loginPage.url);
-    loginPage.login("admin", "admin");
+  I.amOnPage(loginPage.url);
+  loginPage.login('admin', 'admin');
 });
 
-Scenario('Open Remote Instances Page and verify Discovery and Adding AWS RDS MySQL 5.6 instance', async (I, remoteInstancesPage, pmmInventoryPage) => {
-    let instanceIdToMonitor = "rds-mysql56";
+Scenario(
+  'Verify Discovery and adding AWS RDS MySQL 5.6 instance for monitoring @not-pr-pipeline',
```

Comment:
All Scenarios in this test file should be used in the test run, they are related to JS changes.